packages/sources/src/addSourceWizard/ValidatorReset.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import useFormApi from '@data-driven-forms/react-form-renderer/dist/cjs/use-form-api';
+
+const ValidatorReset = ({ name }) => {
```

Comment:
Can you add a link to the issues so everybody knows why this abomination has to exist? :smile: